aiosmtpd/tests/conftest.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+import asyncio
+import inspect
+
+from aiosmtpd.controller import Controller
+from aiosmtpd.handlers import Sink
+from contextlib import suppress
+from smtplib import SMTP as SMTPClient
+from ..testing.helpers import DecodingController
+from typing import Any, Callable, Dict, NamedTuple, Optional, Type
+
+
+class HostPort(NamedTuple):
+    host: str = "localhost"
+    port: int = 8025
+
+
+class Global:
+    SrvAddr: HostPort = HostPort()
+
+    @classmethod
+    def set_addr_from(cls, contr: Controller):
+        cls.SrvAddr = HostPort(contr.hostname, contr.port)
```

Comment:
The class `Global` is used to feed the hostname:port of the Controller to the SMTP Client.